packages/nextra/src/client/components/index.ts

```diff
@@ -15,3 +15,4 @@ export { Tr } from './tr.js'
 export { Mermaid } from '@theguild/remark-mermaid/mermaid'
 export { MathJax, MathJaxContext } from 'better-react-mathjax'
 export { Playground } from './playground.js'
+export { Popup, PopupTrigger, PopupPanel } from './popup.js'
```

Comment:
since all tree components designed to use together, can you refactor them to
```js
Popup
Popup.Trigger
Popup.Panel
```

in `popup.tsx` you need to do the following at the end of file:

```jsx
export const Popup = Object.assign(_Popup, {
  Trigger,
  Pannel
})
```